contracts/deployables/erc20/VotesERC20V1.sol

```diff
@@ -14,14 +15,16 @@ import {ERC20PermitUpgradeable} from "@openzeppelin/contracts-upgradeable/token/
 import {VotesUpgradeable} from "@openzeppelin/contracts-upgradeable/governance/utils/VotesUpgradeable.sol";
 import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
 
 contract VotesERC20V1 is
     IVotesERC20V1,
     Version,
     ERC20VotesUpgradeable,
     ERC20PermitUpgradeable,
     UUPSUpgradeable,
-    Ownable2StepUpgradeable
+    Ownable2StepUpgradeable,
+    AccessControlUpgradeable
```

Comment:
Do we need AccessControl inherited in this contract? Doesn't look like we're using it here, and could just inherit it in VotesERC20LockableV1.sol

Reply:
Maybe we could update Version to remove ERC165, similar to what @adamgall did in [this comment](https://github.com/decentdao/decent-contracts/pull/314#issuecomment-2937679873)?